### DISCUSSION TITLE
Cancel automatic approval of translations

### DISCUSSION THREAD

**Discussion by yangyangdaji** [GradienceTeam/Gradience#630] [Ideas]
weblate approves translations. It will set the string to be read, so if there is a problem with the translation, you can only change the translation file
In future work, no approval is needed. 
so that it can be edited at any time
![Screenshot_20221012_113735_edit_17368887264014](https://user-images.githubusercontent.com/108261729/195245574-54e53a83-bade-4113-be48-8c4cd8797505.jpg)

![Screenshot_20221012_120839_edit_19017611134075](https://user-images.githubusercontent.com/108261729/195248025-1beb87b3-f39e-4c54-83f6-8592c32e5d75.jpg)

**Comment by yangyangdaji** [comment]
By the way I am discussing the "Approval translation" in weblate at https://github.com/WeblateOrg/weblate/issues/8198 and have added the box or renamed it to avoid misunderstandings if you have suggestions

**Comment by yangyangdaji** [comment]
cc @0xMRTT @daudix-UFO

**Reply by 0xMRTT** [reply]
@daudix-UFO I agree, what about you?

**Reply by 0xMRTT** [reply]
Hey, I've edited all strings, they should all be editable

**Reply by 0xMRTT** [reply]
Also, if you need, you can join the matrix room or the discord server

**Comment by yangyangdaji** [comment]
Okay now Thank you @0xMRTT
